core/app/models/spree/cms_page.rb

```diff
@@ -0,0 +1,64 @@
+module Spree
+  class CmsPage < Spree::Base
+    include Spree::DisplayLink
```

Comment:
```suggestion
    include SingleStoreResource
    include Spree::DisplayLink
```

Reply:
This is done, I don't know why it isn't  picking up as changed.